common/app/routes/Settings/utils/legacyProjectData.js

```diff
@@ -0,0 +1,88 @@
+const legacyFrontEndProjects = {
```

Comment:
Is this only used in `common/app/entites`? We should move it in there instead. This seems like something we might also use in the rest of the app in the future, unless I'm mistaken. If so we should instead move it to `common/utils` to share between server/client.